packages/protocol/contracts/governance/test/MockBondedDeposits.sol

```diff
@@ -40,7 +45,16 @@ contract MockBondedDeposits is IBondedDeposits {
   }
 
   function setWeight(address account, uint256 weight) external {
-    weights[account] = weight;
+    uint256 difference;
+    if (weight >= weights[account]) {
```

Comment:
Consider just adding a `setTotalWeight` function rather than complicating this logic

Reply:
This would require every `setWeight` call in Governance tests to be replaced with a coupled `setWeight` and `setTotalWeight` call. I agree that abstracting away the total weight logic is preferable, but in your opinion, is it worth the added complexity on the test side?